Documentation/Index.md

```diff
@@ -1178,12 +1178,12 @@ Any object that can be encoded and decoded can be stored as a blob of data in SQ
 We can create an `NSData` bridge rather trivially.
 
 ``` swift
-extension NSData: Value {
+extension NSData{
```

Comment:
Yeah it looks like it's because it's already implemented: https://github.com/stephencelis/SQLite.swift/blob/master/SQLite/Foundation.swift#L27

I think the right fix is to actually update the docs to point to existing implementations, and leave the UIImage example for when you need to add your own custom ones.